craco.config.js

```diff
@@ -1,18 +1,19 @@
-const CracoLessPlugin = require('craco-less');
-
+const CracoAntDesignPlugin = require('craco-antd');
+process.env.BROWSER = "none";
```

Comment:
❓ Why is this required?

Reply:
This is required so nodemon doesnt constantly open a new tab in the browser every time it detects a change